common/src/main/java/com/genexus/LocalUtil.java

```diff
@@ -1083,6 +1083,10 @@ public Date ctotex(String date, int format)
 		}
 		catch (ParseException e)
 		{
+			//When parsing a date gives a ParseException we try with setLenient(true) to parse dates only with dates with
+			//daylighttime changes at 00:00 AM (Issue: 93038)
+			if (!isNullTimeValue(date.substring(date.indexOf(' ') + 1), false))
```

Comment:
Is it possible that the expression: date.substring(date.indexOf(' ') + 1) throws a IndexOutOfBoundsException?
According to [substring](https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#substring(int)).
P.e if date is empty, it results in date.substring(1)) which throws IndexOutOfBoundsException.